teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age:, specialization:, name: 'Unknown', parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Kindly make sure you use the correct syntax while initializing the param of your class. as you may notice how it's impossible to create the instance of your classes in the irb
```suggestion
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

Reply:
thank you for you progressive feedback once again! 💯